libs/libmaxminddb/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt.org unless you're affliated. Read https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md

Reply:
I replaced that with proper copyright. :man_facepalming: